src/applications/disability-benefits/526EZ/reference-helpers.js

```diff
@@ -1,8 +1,841 @@
+  4420: 'Myeloma',
+  4430: 'Myeloma, Multiple',
+  4440: 'Myocardial Infarction (MI)',
+  4450: 'Myocarditis',
+  4460: 'Myositis',
+  4470: 'Myxosarcoma',
+  4480: 'Nausea',
+  4490: 'Neck Condition',
+  4500: 'Neck Sprain',
+  4510: 'Neck Strain',
+  4520: 'Neoplasm(s), Benign',
+  4530: 'Neoplasm(s), Malignant',
+  4540: 'Nephritis',
+  4550: 'Nerve Damage',
+  4560: 'Nerve Deafness',
+  4570: 'Nerves',
+  4580: 'Nervous System Disease',
+  4590: 'Nervousness',
+  4600: 'Neuralgia',
+  4610: 'Neuritis',
+  4620: 'Neurologic Signs or Symptoms',
+  4630: 'Neurological Disorder',
+  4640: 'Neuropathy, Peripheral',
+  4650: 'Neuropsychological Signs or Symptoms',
+  4660: 'New Growths, Malignant',
+  4670: 'Niacin Deficiency',
+  4680: 'Non-Hodgkin’s Lymphoma (NHL)',
+  4690: 'Non-Specific Digestive Complaints',
+  4700: 'Non-Specific Ear Condition',
+  4710: 'Non-Specific Genitourinary Complaints',
+  4720: 'Non-Specific Gynecological Complaints',
+  4730: 'Non-Specific Nervous Complaints',
+  4740: 'Non-Specific Neurological Complaints',
+  4750: 'Non-Specific Orthopedic Claims',
+  4760: 'Non-Specific Respiratory Condition',
+  4770: 'Nose Condition',
+  4780: 'Nutritional Deficiency',
+  4790: 'OBS',
+  4800: 'Onchocerciasis',
+  4810: 'Oophoritis',
+  4820: 'Optic Atrophy Associated with Malnutrition',
+  4830: 'Organic Brain Syndrome',
+  4840: 'Organic Diseases of the Nervous System',
+  4850: 'Organic Heart Disease',
+  4860: 'Oroya Fever',
+  4870: 'Orthopedic Condition',
+  4880: 'Osgood Schlatter Disease',
+  4890: 'Osteitis Deformans (Paget’s Disease)',
+  4900: 'Osteoarthritis, Post-Traumatic',
+  4910: 'Osteomalacia',
+  4920: 'Otitis Externa',
+  5000: 'Otitis Media',
+  5010: 'Otosclerosis',
+  5020: 'Ovarian Condition, Bilateral',
+  5030: 'Ovary (Disease, Injury, or Adhesions of)',
+  5040: 'Ovary (Removal of)',
+  5050: 'Painful Joints',
+  5060: 'Palsy',
+  5070: 'Palsy, Bulbar',
+  5080: 'Pancreatitis',
+  5090: 'Paralysis',
+  5100: 'Paralysis Agitans',
+  5110: 'Paralysis, Nerve',
+  5120: 'Paranoid Reaction',
+  5130: 'Parasites',
+  5140: 'Parathyroid Adenoma',
+  5150: 'Parenchymal Lung Disease',
+  5160: 'Parkinson’s Disease',
+  5170: 'Pellagra',
+  5180: 'Penile Condition',
+  5190: 'Peptic Ulcer Disease',
+  5200: 'Pericarditis',
+  5210: 'Periodontal Disease (Pyorrhea)',
+  5220: 'Periostitis',
+  5230: 'Peripheral Neuropathy',
+  5240: 'Peripheral Neuropathy, Acute and Subacute',
+  5250: 'Peripheral Vascular Disease',
+  5260: 'Peritonitis',
+  5270: 'Personality Disorder',
+  5280: 'Petit Mal Epilepsy',
+  5290: 'Pharyngeal Condition',
+  5300: 'Phlebitis',
+  5310: 'Phobias',
+  5320: 'Pinta',
+  5330: 'Plague',
+  5340: 'Plantar Fasciitis',
+  5350: 'Pleural Effusions',
+  5360: 'Pleural Plaques',
+  5370: 'Pleurisy',
+  5380: 'Pneumocystis Carini Pneumonia',
+  5390: 'Pneumonia',
+  5400: 'Polymyositis',
+  5410: 'Porphyria Cutanea Tarda  (PCT) (a Skin Condition)',
```

Comment:
Is there an extra space here? between `Tarda` and `(PCT)`?

Reply:
Oooh, good eye!! :female_detective: